dali/python/nvidia/dali/backend.py

```diff
@@ -47,3 +47,7 @@ def deprecation_warning(what):
 
     for lib in default_plugins:
         LoadLibrary(os.path.join(os.path.dirname(__file__), lib))
+
+if CheckCudaVersion() == -1:
+    deprecation_warning("No CUDA installation detected. Please make sure it is available "
```

Comment:
Should it really be just warning?

Reply:
Yes. DALI can run without GPU if you use only CPU ops and device_id is None.